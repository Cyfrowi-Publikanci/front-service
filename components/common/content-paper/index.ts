export { ContentPaper } from './content-paper';
