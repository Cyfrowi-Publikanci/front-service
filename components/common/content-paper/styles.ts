import { makeStyles } from '@material-ui/core/styles';

export const useStyles = makeStyles(theme => ({
  paperContainer: {
    padding: theme.spacing(3),
    margin: `${theme.spacing(2)}px 0`,
  },
}));
