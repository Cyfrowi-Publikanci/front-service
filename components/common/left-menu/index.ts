export { LeftMenu } from './left-menu';
export { MenuItem } from './menu-item';