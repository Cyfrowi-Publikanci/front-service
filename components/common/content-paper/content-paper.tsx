import { ReactNode } from 'react';
import Paper from '@material-ui/core/Paper';

import { useStyles } from './styles';

interface Props {
  children: ReactNode;
  className?: string;
}

export const ContentPaper = ({ children, className }: Props) => {
  const classes = useStyles();

  return (
    <Paper elevation={3} className={`${classes.paperContainer} ${className || ''}`}>
      {children}
    </Paper>
  );
};
