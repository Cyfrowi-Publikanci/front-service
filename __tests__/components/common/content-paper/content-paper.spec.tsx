import { shallow } from 'enzyme';

import { ContentPaper } from '../../../../components/common/content-paper';

describe('ContentPaper', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <ContentPaper className="class">
        <div>test</div>
      </ContentPaper>,
    );

    expect(wrapper).toMatchSnapshot();
  });
});
