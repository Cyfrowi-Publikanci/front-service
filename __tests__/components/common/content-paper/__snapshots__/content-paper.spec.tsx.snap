// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentPaper should match snapshot 1`] = `
<WithStyles(ForwardRef(Paper))
  className="makeStyles-paperContainer-1 class"
  elevation={3}
>
  <div>
    test
  </div>
</WithStyles(ForwardRef(Paper))>
`;
